### PR TITLE
Add command for dumping PE file headers

### PR DESCRIPTION
**Pull request by rjoleary**
Signed-off-by: Ryan O'Leary <ryanoleary@google.com>

### CODE REVIEW

File: cmds/pe/pe.go [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+	"debug/pe"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"log"
+)
+
+func main() {
+	flag.Parse()
+	if flag.NArg() != 1 {
```
Comment (rminnich) [inline cmds/pe/pe.go]: awesome. If you can make the change to stdin if no arg, we're good.

File: cmds/pe/pe.go [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+	"encoding/json"
+	"errors"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+)
+
+func openPE() (*pe.File, error) {
+	switch flag.NArg() {
```
Comment (hugelgupf) [inline cmds/pe/pe.go]: Generally not nice to parse flags or use the flag package at all outside of main. You could honestly just do this in main.

File: cmds/pe/pe.go [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+func main() {
+	flag.Parse()
+
+	f, err := openPE()
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer f.Close()
+
+	json, err := json.MarshalIndent(f, "", "\t")
```
Comment (hugelgupf) [inline cmds/pe/pe.go]: Huh, clever. Nice.

I'd avoid calling the variable json, though. The package is already called that.

Comment (rjoleary) [inline cmds/pe/pe.go]: Done!

**Pull request merged**